src/testing_system/internal/usecase/Runner.py

```
@@ -0,0 +1,155 @@
+    def _run_eval(self, q: Question, a: Answer) -> List[MetricValue]:
+        return self.eval.execute(q, a)
+
+    async def _process_worker(self, executor):
+        while True:
+            query = await self.process_queue.get()
+            try:
+                loop = asyncio.get_running_loop()
+                processed = await loop.run_in_executor(executor, self._run_process, query)
+                self.experiment.add_answer(processed)
+                await self.eval_queue.put((query,processed))
+            except Exception as e:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)